test/s3_test_script.py

```diff
@@ -235,7 +235,9 @@ def verbose(verbose, prompt, message):
     if not verbose:
         return
 
-    create_iam_role(src_bucket, dest_bucket)
+    # Commenting out due to codecy issue
+    # Commenting out doesn't seem to make any impact on code flow
```

Comment:
replace above comments (line 238-239) with 
# Call here taking two arguments even if function definition has one, and codacy flagged it.
# Commenting out given no functional need seen and no impact seen on removal